webpack/webpack.config.js

```diff
@@ -55,9 +53,21 @@ module.exports = {
         publicPath: '/',
         historyApiFallback: true,
     },
+    optimization: {
+        minimize: false,
```

Comment:
I would think we would want to minimize for production

Reply:
As of webpack 4, minimize is automatically set to true when in production mode, and false when in development mode, so no need to set the flag anymore.